WPMedia/engine-theme-sdk
# Updated TwitterIcon path

## Description

Update with new Twittericon

## Jira Ticket

- [THEMES-1440](https://arcpublishing.atlassian.net/browse/THEMES-1440)

## Acceptance Criteria

[_copy from ticket_](https://www.figma.com/file/NBjMF1v4mzwv9jkn7NEq9V/Themes-Block-Styling-2.0?type=design&node-id=10811%3A147806&mode=design&t=jG5oCFMUMr3b84Ro-1)

## Test Steps

1. Checkout branch - git checkout THEMES-1440
2. Update dependencies - npm i
3. Run footer-block frok pagebuilder 
4. Check out the Twitter icon in footer-block

<img width="1696" alt="Screenshot 2023-10-24 at 3 08 08 PM" src="https://github.com/WPMedia/engine-theme-sdk/assets/83021791/9e818155-51af-4e72-9b20-14031494ec9b">


## Review Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).


[THEMES-1440]: https://arcpublishing.atlassian.net/browse/THEMES-1440?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
